app/concepts/user_auth/operation/sign_in.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module UserAuth::Operation
+  class SignIn < Trailblazer::Operation
+    step Contract::Build(builder: :build_contract)
+    step Contract::Validate()
+    step :find_user_account
+    step :check_credentials
+    step :generate_auth_token
+
+    def build_contract(_ctx, params:, **)
+      UserAuth::Contract::SignIn.new(UserAccount, **params)
+    end
+    
+    def find_user_account(ctx, params:, **)
+      ctx['user_account'] = UserAccount.find_by(email: params[:email])
+      Trailblazer::Activity::Right
```

Comment:
Why right?

Reply:
Because according to tibetan drawings ☯️